tox.ini

```diff
@@ -19,28 +20,22 @@ deps=
     django16: django>=1.6,<1.7
     django17: django>=1.7,<1.8
     django18: django>=1.8,<1.9
+    django19: django>=1.9,<2.0
```

Comment:
Next version will be `1.10.0`

https://www.djangoproject.com/weblog/2015/jun/25/roadmap/